spec/maestro_spec.rb

```diff
@@ -50,7 +50,7 @@ def json_response
 
   describe "Valid command" do
     context "command with no args" do
-      let(:text) { "stop" }
+      let(:text) { "Stop" }
```

Comment:
I'm guessing this is meant to test the case sensitivity. Can you add a comment that says that? It might be unclear to future devs looking at this.